jardin/instrumentation/event.py

```diff
@@ -0,0 +1,10 @@
+from typing import *
+from dataclasses import dataclass, field
+
+@dataclass
+class Event:
+  name: str
+  start_time: float = 0.0
+  end_time: float = 0.0
+  duration: float = 0.0
```

Comment:
How is the duration measured? seconds? millis?

Reply:
fractional seconds. I added it to field name.